Modules/CaPTkCommon/include/CaPTkCohortSeries.h

```diff
@@ -0,0 +1,97 @@
+#ifndef CaPTkCohortSeries_h
+#define CaPTkCohortSeries_h
+
+// The following header file is generated by CMake and thus it's located in
+// the build directory. It provides an export macro for classes and functions
+// that you want to be part of the public interface of your module.
+#include <MitkCaPTkCommonExports.h>
+
+namespace captk
+{
+class CohortImage;
+}
+
+#include "mitkImage.h"
+#include "mitkLabelSetImage.h"
+
+#include <QString>
+#include <QList>
+#include <QSharedPointer>
+
+namespace captk
+{
+/** \class CohortSeries
+ * \brief Defines information about a series
+ * and holds the series's various images
+ * 
+ * Note: series is a combination of modality and 
+ * acquisition protocol. Multiple series can happen
+ * in a study.
+ */
+class MITKCAPTKCOMMON_EXPORT CohortSeries
```

Comment:
Do you want to call these CohortSeries, CohortStudy, etc.? Or just Series, Study,etc.

Reply:
I like the "Cohort" prefix because it shows that all belong to the same family